libeos-payg/multi-task.c

```diff
@@ -39,12 +39,27 @@ epg_multi_task_set (GTask *task,
                       GUINT_TO_POINTER (pending));
 }
 
+/**
+ * epg_multi_task_attach:
+ * @task: (transfer none): the task
+ * @pending: initial number of pending operations for @task.
+ *
+ * Initializes @task as having @pending operations pending.  The number of
+ * pending operations can be incremented with epg_multi_task_increment(), and
+ * decremented with epg_multi_task_return_boolean(),
+ * epg_multi_task_return_pointer(), and epg_multi_task_return_error().
+ *
+ * The number of pending operations is stored by occult means, leaving the
```

Comment:
More documentation should be like this. ⛧

Reply:
Credit where credit's due: I stole this phrase from a [New Yorker review of Black Mirror](https://www.newyorker.com/magazine/2015/01/05/button-pusher) on the occasion of its legal release in the US:

> For a full day after watching the first episode (which I obtained through occult means, before Netﬂix made the show available to U.S. viewers), I felt disoriented, dropped on a new planet.